### PR TITLE
Fix issue where VF stops working after changing device while muted

### PR DESCRIPTION
**Pull request by LeviSklaroff**
**Issue #:**
- Voice Focus stops working if a user mutes and changes their device
**Description of changes:**
- Mutes the transform device instead of the original device
**Testing:**
- Tested that voice focus now works under this scenario.
*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
Yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/devicecontroller/DefaultDeviceController.ts [hunk 1]
```diff
@@ -687,7 +687,12 @@ export default class DefaultDeviceController
       return;
     }
     let isChanged = false;
-    for (const track of this.activeDevices['audio'].stream.getTracks()) {
+    let audioStream = this.activeDevices['audio'].stream 
+    if (this.useWebAudio) {
```
Comment (xuesichao) [inline src/devicecontroller/DefaultDeviceController.ts:691]: Do we need to add unit test to cover this if condition?

Comment (LeviSklaroff) [reply src/devicecontroller/DefaultDeviceController.ts]: We already have tests that trigger this condition

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fix issue where VF stops working after changing device while muted
```
Comment (xuesichao) [inline CHANGELOG.md:26]: "VF" -> "Amazon Voice Focus", and add comma end of line.

File: CHANGELOG.md [hunk 3]
```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Fixed an issue for mute local when there is no audio input.
 - Fixed trucation of video subscriptions not occuring if the resubscribe was driven by `MonitorTask`.
```
Comment (xuesichao) [inline CHANGELOG.md:28]: I would appreciate if you could update these two "Fixed" to "Fix".

File: CHANGELOG.md (hunk 2 above)
Comment (LeviSklaroff) [reply CHANGELOG.md]: Changed

File: CHANGELOG.md (hunk 3 above)
Comment (LeviSklaroff) [reply CHANGELOG.md]: Fixed

**Review by xuesichao — APPROVED** [review]

**Review by xuesichao — APPROVED** [review]

**Pull request merged**